integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/entity/DatabaseEntity.java

```diff
@@ -1,4 +1,4 @@
-package com.sequenceiq.it.cloudbreak.newway;
+package com.sequenceiq.it.cloudbreak.newway.entity;
```

Comment:
please create a database package inside entity and move the database related entities there.